website/docs/providers/index.html.markdown

```diff
@@ -20,6 +20,8 @@ browse its documentation.
 Provider documentation in the Registry is versioned; you can use the version
 menu in the header to change which version you're viewing.
 
+Learn more about provider documentation rendering on the [provider registry documentation](https://www.terraform.io/docs/registry/providers/docs.html).
```

Comment:
- Markup style: internal links should omit hostname and protocol, so they work similarly in prod and in local previews. 
- Let's make that callout even more general, since there's a lot of helpful stuff over there!

```suggestion
Learn more about writing, generating, and rendering provider documentation
in the [provider publishing documentation](/docs/registry/providers/docs.html).
```